src/DssSpell.sol

```diff
@@ -38,94 +119,252 @@ contract DssSpellAction is DssAction {
+        // CHAINLINK | 1,500 DAI/day | 3 years | Vest Target: 0xfB5e1D841BDA584Af789bDFABe3c6419140EC065
+        // TECHOPS   | 1,000 DAI/day | 1 years | Vest Target: 0x5A6007d17302238D63aB21407FF600a67765f982
 
-        // Mip: https://mips.makerdao.com/mips/details/MIP107#7-1-multichain-engineering-budget
-        // Multichain Engineering Budget          | 2023-05-01 00:00:00 to 2024-04-30 23:59:59 | 2,300,000 DAI | 0x868B44e8191A2574334deB8E7efA38910df941FA
 
-        // --------- Data Insights MKR Transfer ---------
-        // Mip: https://mips.makerdao.com/mips/details/MIP40c3SP64#mkr-vesting
+        // --------- CAIS Bootstrap Funding ---------
+        // Poll: https://vote.makerdao.com/polling/Qmc6Wqrc#poll-detail
         // NOTE: ignore in goerli
-        // DIN-001 - 103.16 MKR - 0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F
+
+
+        // --------- Onboard GNO to Spark ---------
+        // Poll: https://vote.makerdao.com/polling/QmXdGdxS#poll-detail
+        // Forum: https://forum.makerdao.com/t/onboarding-of-gno-to-spark/20831
+        // List of addresses: https://github.com/marsfoundation/sparklend/blob/master/script/output/5/spark-latest.json
+        {
+            // Whitelist the GNO Fig adapter
+            MedianAbstract(GNO_MEDIANIZER).kiss(SPARK_GNO_ORACLE);
+
+            // Set DAI as a borrowable asset in isolation mode
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setBorrowableInIsolation(MCD_DAI, true);
+
+            // Add GNO
+            address token = DssExecLib.getChangelogAddress("GNO");
+            PoolConfiguratorLike.InitReserveInput[] memory input = new PoolConfiguratorLike.InitReserveInput[](1);
+            input[0] = PoolConfiguratorLike.InitReserveInput({
+                aTokenImpl: SPARK_ATOKEN_IMPL,
+                stableDebtTokenImpl: SPARK_STABLE_DEBT_TOKEN_IMPL,
+                variableDebtTokenImpl: SPARK_VARIABLE_DEBT_TOKEN_IMPL,
+                underlyingAssetDecimals: GemAbstract(token).decimals(),
+                interestRateStrategyAddress: SPARK_INTEREST_RATE_STRATEGY,      // Dummy strategy - compare to other borrow-disabled asset like sDAI
+                underlyingAsset: token,
+                treasury: SPARK_TREASURY,
+                incentivesController: address(0),
+                aTokenName: "Spark GNO",
+                aTokenSymbol: "spGNO",
+                variableDebtTokenName: "Spark Variable Debt GNO",
+                variableDebtTokenSymbol: "variableDebtGNO",
+                stableDebtTokenName: "Spark Stable Debt GNO",
+                stableDebtTokenSymbol: "stableDebtGNO",
+                params: ""
+            });
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).initReserves(input);
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).configureReserveAsCollateral({
+                asset: token, 
+                ltv: 2000,
+                liquidationThreshold: 2500,
+                liquidationBonus: 11000
```

Comment:
For spell team, this should be read as `1_10_00` or `110_00` and is correct.

I thought it was set to 11 somehow instead of 10% (idk why I couldn't see the 110 here) but [this is correctly set](https://github.com/aave/aave-v3-core/blob/29ff9b9f89af7cd8255231bc5faf26c3ce0fb7ce/contracts/interfaces/IPoolConfigurator.sol#L301)

Reply:
Sure can adjust the format.

Reply:
done in https://github.com/makerdao/spells-goerli/pull/198